framework/src/positions/ElementGroupCentroidPositions.C

```diff
@@ -9,6 +9,8 @@
 
 #include "ElementGroupCentroidPositions.h"
 
+#include "timpi/communicator.h"
```

Comment:
@roystgnr this does not seem fixing the unity build error. Maybe we need a different header like "libmesh/libmesh_call_mpi.h" here?

Reply:
The non-unity build error, you mean?

Ah, I think I see.  `communicator.h` is sufficient for the basic `Communicator::sum` declaration, but depending on what you're summing you might also need `timpi/parallel_implementation.h` (for all the specializations there) and a header defining a `StandardType` and/or `Packing` specialization for the particular type of data you're summing, in this case probably `libmesh/parallel_algebra.h` for `Point`.

Reply:
Trying `libmesh/parallel_algebra.h`. Will see whether it is enough.